### PR TITLE
Github issues templates

### PR DESCRIPTION
**Pull request by strump**
Look like PR #2484 has stuck.
I created simple issues templates: "Bug report" and "Feature request".
Got to my fork https://github.com/strump/organicmaps/ and try to create an issue:

Screenshot 2023-02-02 at 10 09 21

Screenshot 2023-02-01 at 18 13 15 Screenshot 2023-02-01 at 18 13 29

Closes #2484

### CODE REVIEW

**Comment by strump** [comment]
@biodranik I don't think GitHub can detect duplicated issues automatically. Can't google anything about it. But I can add links to some popular issues (Public transport, Track recording, etc) to template description.

**Comment by endim8** [comment]
Maybe we could link to epic issues (styles, routing, UI etc), and link often duplicated issues & sub-epic issues in them?

**Comment by biodranik** [comment]
It's not about duplication, it's about having questions and ideas discussed in GH Discussions instead of polluting issues.

**Comment by Jean-BaptisteC** [comment]
@biodranik Yes, check this https://github.com/streetcomplete/StreetComplete/blob/master/.github/ISSUE_TEMPLATE/config.yml

**Comment by strump** [comment]
Added links to:
* https://github.com/organicmaps/organicmaps/discussions
* https://github.com/organicmaps/organicmaps/blob/master/docs/TRANSLATIONS.md

Screenshot 2023-02-02 at 10 09 21

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve OrganicMaps
```
Comment (biodranik) [inline .github/ISSUE_TEMPLATE/bug_report.md:3]: Describe your issue in details to help us improve Organic Maps

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
```
Comment (biodranik) [inline .github/ISSUE_TEMPLATE/bug_report.md:10]:
```suggestion
**Describe the issue**
```
Not everyone knows what a "bug" is.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+A clear and concise description of what the bug is.
```
Comment (biodranik) [inline .github/ISSUE_TEMPLATE/bug_report.md:11]:
```suggestion
Please write here a clear and concise description of what the bug/issue is about.
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+**Steps To Reproduce**
```
Comment (biodranik) [inline .github/ISSUE_TEMPLATE/bug_report.md:13]:
```suggestion
**Steps to reproduce**
```
Consistency matters! ;-)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots or screen recordings to help explain your problem.
+
+**Application:**
```
Comment (biodranik) [inline .github/ISSUE_TEMPLATE/bug_report.md:25]:
```suggestion
**System information:**
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+ - OS: [iOS, Android, Linux, MacOS, etc]
```
Comment (biodranik) [inline .github/ISSUE_TEMPLATE/bug_report.md:26]:
```suggestion
 - Operating system and its version: [iOS 12, Android 10, Ubuntu 22, MacOS Big Sur, etc.]
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+ - OrganicMaps version: [you can find it clicking "?" button]
```
Comment (biodranik) [inline .github/ISSUE_TEMPLATE/bug_report.md:27]:
```suggestion
 - Organic Maps version: [you can find it by clicking the "?" button]
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+ - Device Model: [e.g. iPhone6, Samsung S22]
+
+**Additional context**
+Add any other context about the problem here.
```
Comment (biodranik) [inline .github/ISSUE_TEMPLATE/bug_report.md:31]:
```suggestion
Please add any other context and important details/notes/comments about the problem here.
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 9]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Discussions
+    url: https://github.com/organicmaps/organicmaps/discussions
+    about: For discussions about the usage of OrganicMaps, asking questions or talking about ideas which are not yet actionable.
+  - name: Translations
+    url: https://github.com/organicmaps/organicmaps/blob/master/docs/TRANSLATIONS.md
+    about: For contributing translations of OrganicMaps
```
Comment (biodranik) [inline .github/ISSUE_TEMPLATE/config.yml:8]:
```suggestion
    about: Translate Organic Maps into your language
- name: News
    url: https://organicmaps.app/news/
    about: Check the latest project news
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 10]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Discussions
+    url: https://github.com/organicmaps/organicmaps/discussions
+    about: For discussions about the usage of OrganicMaps, asking questions or talking about ideas which are not yet actionable.
```
Comment (biodranik) [inline .github/ISSUE_TEMPLATE/config.yml:5]:
```suggestion
    about: Discuss the usage of OrganicMaps, ask questions, or talk about ideas that are not yet actionable.
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 11]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for OrganicMaps
+title: ''
+labels: ''
```
Comment (biodranik) [inline .github/ISSUE_TEMPLATE/feature_request.md:5]:
```suggestion
labels: [Enhancement]
```
Is it the correct format?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 12]
```diff
@@ -0,0 +1,20 @@
...
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```
Comment (biodranik) [inline .github/ISSUE_TEMPLATE/feature_request.md:11]:
```suggestion
A clear and concise description of what the problem is. For example:
I'm always frustrated when [...]
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 13]
```diff
@@ -0,0 +1,20 @@
...
+
+**Describe the solution you'd like**
```
Comment (biodranik) [inline .github/ISSUE_TEMPLATE/feature_request.md:13]:
```suggestion
**Describe the solution you would like**
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 14]
```diff
@@ -0,0 +1,20 @@
...
+A clear and concise description of what you want to happen.
```
Comment (biodranik) [inline .github/ISSUE_TEMPLATE/feature_request.md:14]:
```suggestion
A clear and concise description of what you want to see in Organic Maps.
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 15]
```diff
@@ -0,0 +1,20 @@
...
+
+**Describe alternatives you've considered**
```
Comment (biodranik) [inline .github/ISSUE_TEMPLATE/feature_request.md:16]:
```suggestion
**Describe alternatives you have considered**
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 16]
```diff
@@ -0,0 +1,20 @@
...
+A clear and concise description of any alternative solutions or features you've considered.
```
Comment (biodranik) [inline .github/ISSUE_TEMPLATE/feature_request.md:17]:
```suggestion
- How do you solve this issue now with Organic Maps or other apps?
- Attach any examples, screenshots, or screen recordings from other apps that help us to better understand the idea.
```

**Review by biodranik — CHANGES_REQUESTED** [review]
Thanks!

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (strump) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Fixed

**Review by biodranik — APPROVED** [review]
Thanks! Let's see how it will work )

**Pull request merged**